### DISCUSSION TITLE
Holograms Not Working On 1.7.10

### DISCUSSION THREAD

**Discussion by RupticMC** [Zrips/CMI#3866] [Feature Requests]
**Description of issue:**

--- Holograms are not showing up in game and no errors when creating the holograms. Saw some errors on startup but not really any that I see that pops out that regards to holograms (maybe).

**ERROR (DELETE IF YOU HAVE NO ERROR):**

--- Errors that I saw on startup: https://paste.helpch.at/okukojenan.m
Lastest.log: https://pastebin.com/qzC9PVHF

```
######################
## REPLACE WITH ERROR ##
######################
```

**Cmi Version (using`/cmi version`):** 
8.7.5.0
**Server Type (Spigot/Paperspigot/etc):** 
UraniumMC (Basically a better version of Thermos/Cauldron for modded 1.7.10) Uses Spigot...
Uranium version 1710-dev-5-B271-31d6587
**Server Version (using `/ver`):** 
dev 1.7.10-R0.1-SNAPSHOT

Comment:

--- I know it is a fork, but I know an older version of Holographic Displays works perfectly fine because it was meant for 1.7.10 (ver 2.1.12). It does not work with the newer versions of the plugin because 1.8+ holograms are different than 1.7.10... Using a different entity or something of the sort (saw it somewhere). I assume the same thing is happening with CMI. The comment that I am linking below will explain what I am talking about between versions.

Not sure if you are able to add this 1.7.10 support or have an option in the config to use 1.7.10 holograms. Thank you.

https://github.com/ProtocolSupport/ProtocolSupport/issues/129#issuecomment-175396528

**Comment by Zrips** [comment]
This is not a bug in particular as there never been a support for holograms before 1.8 version. 1.7.10 has completely different handling of it and it would require quite allot of changes to be done to make it work, some what, as i cant even add every feature holograms currently have because of some general limitations of this outdated minecraft version. 
1.7.10 is 6 year old version, sooner or later i will be dropping support for it entirely as its too outdated to keep with it. So at this point i can only mark this as possible feature/improvement request.

**Comment by RupticMC** [comment]
Thank you. I will be working with a team of sort to create the mod packs that I currently use to new ones so I can transition to 1.12.2 much quicker. I have to move to newer versions sooner or later. Guess this was the push I needed. :)

**Comment by Zrips** [comment]
@RupticMC 
Yea, upgrading to atleast 1.12.2 will give you quite few new features to play around and you will have allot more up to date plugins supporting that.

**Comment by RupticMC** [comment]
It's just that when doing this, it takes time to create our own mod packs to replace the current 1.7.10 ones almost exactly. It will also really hurt my player count for a bit.

Anyways, thank you.
